File: Python solutions/2433. Find The Original Array of Prefix Xor.py
class Solution(object):
    def findArray(self, pref):
        """
        :type pref: List[int]
        :rtype: List[int]
        """
        ans=[]
        ans.append(pref[0])
        for i in range(1, len(pref)):
            ans.append(pref[i-1]^pref[i])      
        return ans

File: Go lang/2433. Find The Original Array of Prefix Xor.go
func findArray(pref []int) []int {
    ans := make([]int, len(pref))
	ans[0] = pref[0]
	for i := 1; i < len(pref); i++ {
        ans[i] = pref[i-1] ^ pref[i]
    }
	return ans
}